docs/BR.md

```diff
@@ -1693,8 +1693,13 @@ The CA SHALL reject a certificate request if one or more of the following condit
 1. The Key Pair does not meet the requirements set forth in [Section 6.1.5](#615-key-sizes) and/or [Section 6.1.6](#616-public-key-parameters-generation-and-quality-checking);
 2. There is clear evidence that the specific method used to generate the Private Key was flawed;
 3. The CA is aware of a demonstrated or proven method that exposes the Applicant's Private Key to compromise;
-4. The CA has previously been made aware that the Applicant's Private Key has suffered a Key Compromise, such as through the provisions of [Section 4.9.1.1](#4911-reasons-for-revoking-a-subscriber-certificate);
-5. The CA is aware of a demonstrated or proven method to easily compute the Applicant's Private Key based on the Public Key (such as a Debian weak key, see <https://wiki.debian.org/SSLkeys>).
+4. The CA has previously been notified that the Applicant's Private Key has suffered a Key Compromise using the CA's procedure for revocation request as described in [Section 4.9.3](#493-procedure-for-revocation-request) and [Section 4.9.12](4912-special-requirements-re-key-compromise);
+5. The Public Key corresponds to an industry-demonstrated weak Private Key. For requests submitted on or after November 15, 2024, at least the following precautions SHALL be implemented:
+    1. The CA SHALL reject Debian weak keys (https://wiki.debian.org/SSLkeys).
```

Comment:
I know this is going around in circles, but here's one last suggestion for a possible way to update this language. No strong feelings here, also happy to just continue with this language as-is so we can focus on fixing Debian in the future instead.
```suggestion
    1. In the case of Debian Weak Keys (https://wiki.debian.org/SSLkeys), the CA SHALL reject keys identified by https://github.com/badkeys/debianopenssl or equivalent.
```